prime-angular-frontend/src/app/modules/provision/pages/enrolments/enrolments.component.html

```diff
@@ -0,0 +1,115 @@
+<h1 class="h2 my-4">Pharmacist Enrolments</h1>
+
+<div class="row justify-content-end">
+  <div class="col text-right">
+
+    <mat-form-field>
+      <mat-label>Filter By Status</mat-label>
+      <mat-select value=""
+                  (selectionChange)="filterByStatus($event)">
+        <mat-option value="">All</mat-option>
+        <mat-option *ngFor="let status of statuses"
+                    [value]="status.code">
+          {{ status.name }}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+
+  </div>
+</div>
+
+<table mat-table
+       [dataSource]="dataSource"
+       class="w-100">
+
+  <!-- Applied Date Column -->
+  <ng-container matColumnDef="appliedDate">
+    <th mat-header-cell
+        *matHeaderCellDef> Application Date </th>
+    <td mat-cell
+        *matCellDef="let enrolment"> {{ enrolment.appliedDate | date: 'MMM d, y' }} </td>
```

Comment:
Can we constant the date formatters? Doesn't have to be now, but maybe on our backlog.

Reply:
Definitely, I can make our own DatePipe and have it set to any default format easily.  The Angular DatePipe has lots of formatting options, but has no global default I can set.

Reply:
Definitely, I can make our own DatePipe and have it set to any default format easily.  The Angular DatePipe has lots of formatting options, but has no global default I can set.